analytics/file.go

```diff
@@ -0,0 +1,60 @@
+package analytics
```

Comment:
I think this is the sample analytics adapter we included to test the platform and demo how to make an adapter. Please add comments. Lots of them.